cmd/customer-admin-controller/customer-admin-controller.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"runtime"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/openshift/openshift-azure/pkg/controllernamespace"
+	"github.com/openshift/openshift-azure/pkg/controllerrolebinding"
+)
+
+func printVersion() {
```

Comment:
I'm far more interested in printing gitCommit like all the other tools do.  Can you just do that instead?

Reply:
Added commit like the sync pod.